pkgs/development/compilers/elm/packages/ghc9_2/elm-format/elm-format-test-lib.nix

```diff
@@ -1,6 +1,20 @@
-{ mkDerivation, avh4-lib, base, containers, fetchgit, filepath
-, hspec, hspec-core, hspec-golden, lib, mtl, split, tasty
-, tasty-discover, tasty-hspec, tasty-hunit, text
+{ mkDerivation
+, avh4-lib
+, base
+, containers
+, fetchgit
+, filepath
+, hspec
+, hspec-core
+, hspec-golden
+, lib
+, mtl
+, split
+, tasty
+, tasty-discover
+, tasty-hspec
+, tasty-hunit
+, text
 }:
```

Comment:
these files are generated by cabal2nix and are not meant to be edited by hand. What did you use to format them? If you did this by hand than I think we should rollback the change as it will be undone next time script we run the code gen.

Reply:
I should be able to rollback, it was mistakenly formatted indeed.